tests/cmdline/commands/test_computer.py

```diff
@@ -511,6 +514,70 @@ def test_show(self):
         assert '--username=' in result.output
         assert result_cur.output == result.output
 
+    def test_computer_export_setup(self):
+        """Test if 'verdi computer export setup' command works"""
+        self.comp_builder.label = 'test_computer_export_setup'
+        self.comp_builder.transport = 'core.ssh'
+        comp = self.comp_builder.new()
+        comp.store()
+
+        exported_setup_filename = 'computer-setup.yml'
+        try:
+            result = self.cli_runner(computer_export_setup, [comp.label, exported_setup_filename])
+            assert 'Success' in result.output, 'Command should have run successfull.'
+            assert (
+                f'{exported_setup_filename}' in result.output
+            ), 'Filename should be in terminal output but was not found.'
+            assert os.path.exists(
+                exported_setup_filename
+            ), f"'{exported_setup_filename}' was not created during export."
+
+            # verifying correctness by comparing internal and loaded yml object
+            with open(exported_setup_filename, 'r', encoding='utf-8') as yfhandle:
+                configure_setup_data = yaml.safe_load(yfhandle)
+            assert configure_setup_data == self.comp_builder.get_computer_spec(
+                comp
+            ), 'Internal computer configuration does not agree with exported one.'
+        except Exception:
+            raise
+        finally:
+            if os.path.exists(exported_setup_filename):
+                os.remove(exported_setup_filename)
```

Comment:
For tests, it is simpler to use the `tmp_path` fixture that comes with `pytest`. This will automatically take care of the cleanup so you don't have to manually `try/except/finally`.

Reply:
done